app/controllers/orders_controller.rb

```diff
@@ -59,6 +59,20 @@ def order_cancel
     @payment.update(status: payment_status)
   end
 
+  def index
+    @orders = @current_user.orders.where.not(status: 'cart').includes(:line_items).order(id: :desc)
```

Comment:
Is this Frontend or Backend?
If backend then we should show cart orders as well

Reply:
It is frontend